lib/chef/knife/core/bootstrap_context.rb

```diff
@@ -164,7 +165,12 @@ def latest_current_chef_version_string
 
         def first_boot
           (@config[:first_boot_attributes] || {}).tap do |attributes|
-            attributes.merge!(:run_list => @run_list)
+            if @config[:policy_name] && @config[:policy_group]
```

Comment:
Not strictly necessary, but to me it feels a bit better to check both since we are using both.